<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-9BI5PNMM" failures="1" tests="1" name="Login Test" time="0.11" errors="0" timestamp="2021-07-04T22:22:24 EDT">
  <testcase classname="TestCases.Login" name="login" time="0.027">
    <failure type="java.lang.AssertionError" message="Failed Because of 404 error">
      <![CDATA[java.lang.AssertionError: Failed Because of 404 error
at TestCases.Login.login(Login.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Login Test -->
